components/services/bat_ledger/bat_ledger_app.cc

```diff
@@ -27,9 +27,9 @@ void OnBatLedgerServiceRequest(
 }  // namespace
 
 BatLedgerApp::BatLedgerApp(
-        service_manager::mojom::ServiceRequest request) :
-    service_binding_(this, std::move(request)),
-    service_keepalive_(&service_binding_, base::TimeDelta()) {
+      service_manager::mojom::ServiceRequest request) :
```

Comment:
Please fix all linter changes, i.e. should be 4 spaces here instead of 6

Reply:
fixed